docs/install/install-methods.md

```diff
@@ -13,3 +13,5 @@ target platform, please [raise an issue](https://github.com/vignetteapp/vignette
 - Steam (Coming soon!)
 - [AUR (Linux)](install-aur.md)
 - [Portable](install-portable.md)
+
+<link rel="stylesheet" href="/styles/disableNext.css" />
```

Comment:
Find a way to get this inserted for every page without requiring us to do it manually.

Reply:
As for now, I added a remark plugin so it's simpler and easier to apply the fix. Just use `-(prev|next|both)-` as markdown body and it will replace it.